include/deal.II/base/vectorization.h

```diff
@@ -1620,12 +2106,36 @@ class VectorizedArray<double>
    * 16 bytes, as opposed to casting a double address to
    * VectorizedArray<double>*.
    */
-  void store (double *ptr) const
+  DEAL_II_ALWAYS_INLINE void store (double *ptr) const
   {
     _mm_storeu_pd (ptr, data);
   }
 
   /**
+   * Gather operation.
```

Comment:
Can you add the same documentation here (and for the next function) as you had further up above where you provided the equivalent code in the documentation?